fb_login.py

```diff
@@ -0,0 +1,19 @@
+from selenium import webdriver
+import getpass
+
+print("please enter your username and password for facebook login")

```

Comment:
Please encapsulate the statements that ask the user to enter information, in a function. 

We will try to use DRY practices so that we can have good codebase. I would also recommend that we put all the functions that are used to perform different functions be inside a class.

Please read [this](https://thealphadollar.github.io/learning/2019/05/13/go-dry.html) for more information and approach me if you think anything is confusing. This will help you improve your coding etiquettes and thus give you an edge over others.